fe/fe-core/src/main/java/com/starrocks/persist/gson/GsonUtils.java

```diff
@@ -426,8 +426,11 @@ public <T> TypeAdapter<T> create(Gson gson, TypeToken<T> type) {
             TypeAdapter<T> delegate = gson.getDelegateAdapter(this, type);
 
             return new TypeAdapter<T>() {
-                public void write(JsonWriter out, T value) throws IOException {
-                    delegate.write(out, value);
+                public void write(JsonWriter out, T obj) throws IOException {
```

Comment:
Better to change the PostProcessTypeAdapterFactory to ProcessHookTypeAdapterFactory

Reply:
update